src/main/java/com/squareup/kotlinpoet/KotlinFile.kt

```diff
@@ -148,7 +148,10 @@ class KotlinFile private constructor(builder: KotlinFile.Builder) {
       when (member) {
         is TypeSpec -> member.emit(codeWriter, null)
         is FunSpec -> member.emit(codeWriter, null, setOf(KModifier.PUBLIC))
-        is PropertySpec -> member.emit(codeWriter, setOf(KModifier.PUBLIC))
+        is PropertySpec -> {
```

Comment:
This seems off to me . . . the other members have enough context when emitting to include a delimiter or not. Might be worth actually passing that context into the emit method. Like how TypeSpec’s emit() method supplies an enclosing enum name